src/components/input-field/input-field.tsx

```diff
@@ -25,6 +25,9 @@ import { InputType } from './input-field.types';
 import { ListItem } from '@limetech/lime-elements';
 
 /**
+ * @exampleComponent limel-example-input-field-email
+ * @exampleComponent limel-example-input-field-tel
+ * @exampleComponent limel-example-input-field-url
```

Comment:
Is there a reason why these are before the more basic examples? :slightly_smiling_face:

Reply:
LOL!

Yes, because I wanted them to be easily accessible while developing, and then totally forgot about moving them down.

Good catch! 🙏

Reply:
⚡